pkg/eval/math/math.go

```diff
@@ -0,0 +1,286 @@
+// Package math exposes functionality from Go's math package as an elvish
+// module.
+package math
+
+import (
+	"math"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var e
+//
+// ```elvish
+// $math:e
+// ```
+//
+// The value of E: 2.718281.... This variable is read-only.
```

Comment:
Write *e* with a lower case and italic to be consistent with how it is usually written.

Linking to Wikipedia would also be nice.

Reply:
Done